src/pages/Articles/Read/index.js

```diff
@@ -60,6 +66,36 @@ class Read extends Component {
     </div>
   );
 
+  renderArticleMenu = () => {
+    const { article } = this.props;
+    const user = getCurrentUser();
+    return (
+      user && (article.author.username === user.username)
+        ? null
```

Comment:
You can replace this with `&&` operator

Reply:
No what I meant is replace line 94 with `&&`

Reply:
Here,
I am using conditional rendering:
I check whether the logged in user is the same as the article author in which case I render `null` otherwise I render the `report article` button. Using `&&` in line `74` changes the logic to an undesired outcome. 

```
renderArticleMenu = () => {
    const { article } = this.props;
    const user = getCurrentUser();
    return (
      user && (article.author.username === user.username)
        ? null
        : (
          <div className="row">
            <div className="col s1 offset-s11">
              <a className="dropdown-trigger" data-target="report-article-dropdown">
                <i className="material-icons right">more_vert</i>
              </a>
            </div>
          </div>
        )
    );
  };

```
When I replace `?` with `&&` , it changes the conditions of the rendering to something inconsistent.
<img width="807" alt="screen shot 2018-11-29 at 16 34 52" src="https://user-images.githubusercontent.com/31407254/49225334-d2dbf380-f3f4-11e8-8d74-4123a5b7508c.png">

Reply:
@SrMoffat this is what I mean 
```
renderArticleMenu = () => {
    const { article } = this.props;
    const user = getCurrentUser();
    return (
      user && (article.author.username === user.username)
       && (
          <div className="row">
            <div className="col s1 offset-s11">
              <a className="dropdown-trigger" data-target="report-article-dropdown">
                <i className="material-icons right">more_vert</i>
              </a>
            </div>
          </div>
        )
    );
  };
```

Reply:
I have understood what you were going for, it have refactored it to:
```
renderArticleMenu = () => {
    const { article } = this.props;
    const user = getCurrentUser();
    return (
      user && !(article.author.username === user.username)
       && (
       <div className="row">
         <div className="col s1 offset-s11">
           <a className="dropdown-trigger" data-target="report-article-dropdown">
             <i className="material-icons right">more_vert</i>
           </a>
         </div>
       </div>
       )
    );
  };

```